designs/consolidation-price-improvement-factor.md

```diff
@@ -0,0 +1,350 @@
+# Consolidation Price Improvement Factor
```

Comment:
I'm sure you're aware of this: https://github.com/kubernetes-sigs/karpenter/blob/main/designs/spot-consolidation.md#2-price-improvement-factor, but might be worth referencing as prior art.